modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach error issues
```

Comment:
My view is that the word 'issues' could be dropped from this heading: 'Resolving multi-attach errors'.

Reply:
Agreed and resolved.